src/exporter/exporter.js

```diff
@@ -0,0 +1,101 @@
+const {fetchContentTypeDefinitions, fetchContentObjects} = require('../flotiq-api/flotiq-api');
+const fs = require('fs');
+
+exports.export = async (apiKey, directoryPath) => {
+
+    let contentTypedDefinitionsResponse = await getContentTypeDefinitionsData(apiKey)
+    let directoryNumber = 1;
+    let totalObjects = 0;
+    let totalPages = contentTypedDefinitionsResponse.total_pages;
+
+    let page = 1;
+    while (page <= totalPages) {
+        console.log(`CTD Page: ${page}/${totalPages}`);
+        for (let i = 0; i < contentTypedDefinitionsResponse.data.length; i++) {
+            await saveSchema(contentTypedDefinitionsResponse.data[i], directoryPath, directoryNumber);
+            let countSavedObjects = await saveObjects(apiKey, contentTypedDefinitionsResponse.data[i].name, directoryPath, directoryNumber);
+            totalObjects = totalObjects + countSavedObjects;
+            directoryNumber++;
+        }
+        page++;
+        if (page <= totalPages) {
+            contentTypedDefinitionsResponse = await getContentTypeDefinitionsData(apiKey, page);
+        }
+    }
+
+    let totalTypesDefinition = directoryNumber - 1
+    console.log('\x1b[32mSummary:');
+    console.log(`\x1b[32mTotal content types definitions: ${totalTypesDefinition}`);
+    console.log(`\x1b[32mTotal content objects: ${totalObjects}`);
+
+    return {totalTypesDefinition: totalTypesDefinition, totalObjects: totalObjects};
+}
+
+const saveObjects = async (apiKey, ctdName, directoryPath, directoryNumber) => {
+    let contentObjectsResponse = await fetchContentObjects(apiKey, ctdName)
+    let contentObjectsResponseJson = await contentObjectsResponse.json();
+    let totalPages = contentObjectsResponseJson.total_pages;
+    let coDirectoryNumber = 1;
+    let countSavedObjects = 0;
+    console.log(`Total objects: ${contentObjectsResponseJson.total_count}`)
+
+    let pageCo = 1;
+    while (pageCo <= totalPages) {
+        console.log(`CO: ${ctdName} Page: ${pageCo}/${totalPages}`);
+        for (let i = 0; i < contentObjectsResponseJson.data.length; i++) {
+            let co = await clearObject(contentObjectsResponseJson.data[i])
+            await saveObject(ctdName, co, directoryPath, directoryNumber, coDirectoryNumber);
+            coDirectoryNumber++;
+            countSavedObjects++;
+        }
+        pageCo++;
+        if (pageCo <= totalPages) {
+            contentObjectsResponse = await fetchContentObjects(apiKey, ctdName, pageCo);
+            contentObjectsResponseJson = await contentObjectsResponse.json();
+        }
+    }
+    return countSavedObjects;
+}
+
+const clearObject = async (object) => {
+    delete object.internal;
+    delete object.id;
```

Comment:
I think, we should clear `id` only in Content Definitions (sending `id` in POST /api/v1/internal/contenttype is not allowed)

When exporting and importing content objects, id's are useful, and allowed by POST /api/v1/content/xx endpoints